CONTRIBUTING.md

```diff
@@ -2,14 +2,14 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 **Table of Contents**  *generated with [DocToc](http://doctoc.herokuapp.com/)*
 
-- [Welcome to Quick!](#welcome-to-quick!)
+- [Welcome to Quick!](#welcome-to-quick)
   - [Reporting Bugs](#reporting-bugs)
   - [Building the Project](#building-the-project)
   - [Pull Requests](#pull-requests)
     - [Style Conventions](#style-conventions)
   - [Core Members](#core-members)
-    - [Code of Conduct](#code-of-conduct)
```

Comment:
I guess this line is breaking some kind of rule from the QuickBot? That's weird because this change just came from running `doctoc` like it says to do.

Reply:
Nevermind, this is from changing `CONTRIBUTING.md`. I feel as though the documentation for this is most relevant in there, since it only impacts folks who want to contribute to Quick.